.hooks/pre-push

```diff
@@ -2,6 +2,9 @@
 # exit on error
 set -e
 
+# Properly handle SIGINT (Ctrl/Cmd+C) when running paratest.
+trap 'kill 0' SIGINT
```

Comment:
I'm not sure that this is compatible with linux:
```
jamiras@ubuntu24 pr/2652:~/source/RAWeb$ .hooks/pre-push 
trap: SIGINT: bad trap
```

Reply:
I'm suspicious about whether it would work in Windows either. `trap` is not a valid DOS command (but neither is `printf`).

Reply:
Fired up a Linux VM to confirm the error - though is now moot with latest changes.

> I'm suspicious about whether it would work in Windows either
On Windows, I'm pretty sure this hook has always required Git Bash or some other kind of special shell.